devicetypes/smartthings/zwave-radiator-thermostat.src/zwave-radiator-thermostat.groovy

```diff
@@ -230,9 +259,23 @@ private secure(cmd) {
 	}
 }
 
+def multiEncap(cmds) {
+	if (zwaveInfo.cc.contains("8F")) {
+		secure(zwave.multiCmdV1.multiCmdEncap().encapsulate(cmds.collect {
+			cmd -> cmd.format()
```

Comment:
Have we used multiencap anywhere else before this? Any problems?

Reply:
I think we haven't, at least I haven't found any example. In this particular case it works like a charm. I haven't really noticed that such CC existed until I got this device, manufacturer stated in manual that using this would extend battery life, so I figured out it's worth trying. I don't know whether it is commonly used in other devices.